src/main/scala/chisel3/simulator/EphemeralSimulator.scala

```diff
@@ -0,0 +1,47 @@
+package chisel3.simulator
+
+import svsim._
+import chisel3.RawModule
+
+/** Provides a simple API for "ephemeral" invocations (where you don't care about the artifacts after the invocation completes) to
+  * simulate Chisel modules. To keep things really simple, it only provides the
```

Comment:
there is still something wrong english-wise with the sentence "To keep things really simple, ..."

Reply:
Oh, maybe this got borked when I committed a suggestion

Reply:
Tweaked the wording